This patch prevent some compile errors

usbtest.c:79:1: error: unknown type name 'u_short'
usbtest.c:93:1: error: unknown type name 'u_short'
usbtest.c:113:10: error: unknown type name 'u_char'
usbtest.c: In function 'int_handler':
usbtest.c:129:2: warning: incompatible implicit declaration of built-in function 'exit' [enabled by default]
usbtest.c: In function 'help':
usbtest.c:138:5: warning: incompatible implicit declaration of built-in function 'exit' [enabled by default]
usbtest.c: In function 'init':
usbtest.c:188:1: error: unknown type name 'u_short'
usbtest.c:189:1: error: unknown type name 'u_char'
usbtest.c: In function 'doinit':
usbtest.c:201:1: error: unknown type name 'u_short'
usbtest.c:202:1: error: unknown type name 'u_char'
usbtest.c:218:4: warning: passing argument 5 of 'usb_rw_eeprom' from incompatible pointer type [enabled by default]
usbtest.c:123:6: note: expected 'char *' but argument is of type 'int *'
usbtest.c: At top level:
usbtest.c:237:10: error: unknown type name 'u_char'
usbtest.c: In function 'io_rdy':
usbtest.c:400:1: error: unknown type name 'fd_set'
usbtest.c:401:16: error: storage size of 'tv' isn't known
usbtest.c: In function 'io_rdy3':
usbtest.c:414:1: error: unknown type name 'fd_set'
usbtest.c:415:16: error: storage size of 'tv' isn't known
usbtest.c: In function 'usb_rw_eeprom':
usbtest.c:454:1: error: unknown type name 'u_short'
usbtest.c:455:1: error: unknown type name 'u_short'
usbtest.c:464:3: warning: incompatible implicit declaration of built-in function 'memcpy' [enabled by default]
usbtest.c:518:3: warning: incompatible implicit declaration of built-in function 'memcpy' [enabled by default]
usbtest.c: At top level:
usbtest.c:577:6: warning: conflicting types for 'recv3' [enabled by default]
usbtest.c:181:10: note: previous implicit declaration of 'recv3' was here

Index: TSC10LD_V213/usb26/usbtest.c
===================================================================
--- TSC10LD_V213.orig/usb26/usbtest.c
+++ TSC10LD_V213/usb26/usbtest.c
@@ -1,3 +1,4 @@
+#include <sys/types.h>
 #include <stdio.h>
 #include <signal.h>
 #include <fcntl.h>
